pkg/services/contexthandler/contexthandler.go

```diff
@@ -352,7 +352,7 @@ func (h *ContextHandler) initContextWithAPIKey(reqContext *models.ReqContext) bo
 
 	// update api_key last used date
 	if err := h.apiKeyService.UpdateAPIKeyLastUsedDate(reqContext.Req.Context(), apikey.Id); err != nil {
-		reqContext.JsonApiErr(http.StatusInternalServerError, InvalidAPIKey, errKey)
+		reqContext.JsonApiErr(http.StatusInternalServerError, "Update API key expiry", err)
```

Comment:
two issues here:
- wrong error passed into the JsonApiErr. expected `err` was `errKey`
- `InvalidAPIKey` error message used instead of `Update API key expiry`